src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/store/collector/NodeConfigCollector.java

```diff
@@ -141,6 +146,8 @@ public NodeConfigFlowUnit operate() {
       }
       collectHeapMaxSize(flowUnit);
     }
+
+    addConfigToNodeCache();
```

Comment:
can we update the cache once the metric is received from flowunit ?

Reply:
let us also take this opportunity to remove configResult hashmap from this collector. there is no need to store configs into both node config cache / hashmap

Reply:
Updating it once the metric is received from flowunit. Makes sense to also have configMap as it can be used to collect and add to flow units